pkg/template/api/v1/types.go

```diff
@@ -14,6 +14,9 @@ type Template struct {
 	// Standard object's metadata.
 	kapi.ObjectMeta `json:"metadata,omitempty"`
 
+	// Optional: An instructional message for the user processing this template (escaped parameters will be substitued).
```

Comment:
Unfortunately, even core k8s did not seem to have a clear convention to follow. 
>> // Optional: Used as the mounted root, rather than the full Ceph tree, default is /
>> Path string `json:"path,omitempty" protobuf:"bytes,2,opt,name=path"`
versus:
>> // Optional: User is the rados user name, default is admin
>> User string `json:"user,omitempty" protobuf:"bytes,3,opt,name=user"`
versus: 
>>  // GenerateName is an optional prefix, used by the server, to generate a unique ...
>> GenerateName string `json:"generateName,omitempty" protobuf:"bytes,2,opt,name=generateName"`